.github/workflows/pre-merge.yaml

```
@@ -35,6 +35,9 @@ jobs:
       - name: Check build health
         run: ./gradlew buildHealth
 
+      - name: Check detekt-gradle-plugin build health
+        run: ./gradlew :detekt-gradle-plugin:buildHealth
```

```suggestion
        run: ./gradlew buildHealth :detekt-gradle-plugin:buildHealth
```

Wouldn't this make it faster (parallelization)?

And, if they can't be merged, wouldn't it be bette/more clear to execute it like this:

```
./gradlew -p detekt-gradle-plugin buildHealth
```

---

Thought it might be easier to see what went wrong if they're separate steps, but forgot about the loss of parallelisation which I agree is more important. Fixed, thanks!